chaincode/ledger_db.go

```diff
@@ -97,13 +97,18 @@ func (db *LedgerDB) KeyExists(key string) (bool, error) {
 func (db *LedgerDB) Put(key string, object interface{}) error {
 	buff, _ := json.Marshal(object)
 
-	if err := db.cc.PutState(key, buff); err != nil {
+	return db.put(key, buff)
+}
+
+// put stores a value in the chaincode db whether it already exists or not
+func (db *LedgerDB) put(key string, value []byte) error {
```

Comment:
I'm not sure it's necessary to introduce this function - but if you prefer to keep to breakdown, what about renaming `put` to `_PutState` to make the intent more clear?